script.js

```diff
@@ -0,0 +1,54 @@
+var word;
+var splitword = [];
+var guess;
+var wguess = 0;
+var playername;
+
+function info(){
+
+  playername = prompt("What is your name?");
+  word = prompt("okay" + " " + playername + " " + "What is your word");
+  console.log(word);
+  splitword = word.split("");
+  console.log(splitword);
+
+  startgame();
+  win();
+}
+
+function startgame(){
+
+guess = prompt("okay, guess a letter");
+var index = splitword.indexOf(guess);
+console.log(index);
+if(index != -1) {
+  alert("Correct Letter");
+  splitword.splice(index , 1);
+  console.log(splitword);
```

Comment:
Can you think of a function that shows the user the currentState of the game, such as a `displayWord` function that shows the word with the missing letters as underscores. Hint, you could reuse the `index` as a key to "fill in the appropriate blank"